rules/windows/process_creation/sysmon_abusing_debug_privilege.yml

```diff
@@ -0,0 +1,43 @@
+title: Abused Debug Privilege by Arbitrary Parent Processes
+id: d522eca2-2973-4391-a3e0-ef0374321dae
+status: experimental
+description: Detection of unusual child processes by different system processes
+references:
+    - https://image.slidesharecdn.com/kheirkhabarovoffzonefinal-181117201458/95/hunting-for-privilege-escalation-in-windows-environment-74-638.jpg
+date: 2020/10/07
+tags:
+    - attack.privilege_escalation
```

Comment:
Hello @semanurguneysu ! Could you please also add the ATT&CK technique to the tags? Thank you!